fv3core/stencils/temperature_adjust.py

```diff
@@ -13,31 +12,42 @@
 
 @gtstencil()
 def compute_pkz_tempadjust(
-    delp: sd, delz: sd, cappa: sd, heat_source: sd, delt: sd, pt: sd, pkz: sd
+    delp: sd,
+    delz: sd,
+    cappa: sd,
+    heat_source: sd,
+    pt: sd,
+    pkz: sd,
+    delt_time_factor: float,
 ):
```

Comment:
- If you understand what this does, can you add a short docstring?
- Minor: can you change these to `FloatField`?